File: Day_2/Q5.java
import java.util.*;
class Q5{
    //Calculating Sum Of Digits
    static int sum_digits(int n){
        int sum=0;
        if(n < 0) n = -n;//Handling Negative Numbers
        while(n>0){
            sum+=n%10;
            n/=10;
        }
        return sum;
    }
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        //Printing Result
        System.out.println("The Sum Of Digits Of "+n+" Is "+sum_digits(n));
        sc.close();
    }
}

File: Day_2/Q5.cpp
#include <iostream>
using namespace std;
//Calculating Sum Of Digits
int sum_digits(int n){
    int sum=0;
    if(n < 0) n = -n;//Handling Negative Numbers
    while(n > 0){
        sum+=n%10;
        n/=10;
    }
    return sum;
}
int main(){
    int n;
    cin >> n;
    //Printing Result
    cout << "The Sum Of Digits Of "<< n << " Is " <<sum_digits(n);
    return 0;
}